app/helpers/application_helper.rb

```diff
@@ -1219,7 +1219,10 @@ def allowed_filter_db?(db)
   end
 
   def miq_tab_header(id, active = nil, options = {}, &block)
-    content_tag(:li, :class => "#{options[:class]} #{active == id ? 'active' : ''}", :id => "#{id}_tab") do
+    content_tag(:li,
+                :class     => "#{options[:class]} #{active == id ? 'active' : ''}",
+                :id        => "#{id}_tab",
+                'ng-click' => "changeAuthTab('#{id}');") do
```

Comment:
@himdel Yeah, the main purpose was to handle tab-specific logic in the angular controller. 
If there are tabs, there will be tab-specific logic, meaning we would *always* have `changeAuthTab` in every angular controller that deals with tabs and hence chances of something breaking probably don't exist.